commit b9182564c716fdfcb39cc77314ba295990dd4312
Author: Adam C. Emerson <aemerson@redhat.com>
Date:   Wed Nov 16 12:38:08 2016 -0500

    dout: Use dout_context
    
    As a transition for g_ceph_context removal, make all dout macros depends
    on a local macro.
    
    Signed-off-by: Adam C. Emerson <aemerson@redhat.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index d068af6a510..8faf6bdf6b1 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -31,6 +31,7 @@
 
 #include "Mgr.h"
 
+#define dout_context g_ceph_context
 #define dout_subsys ceph_subsys_mgr
 #undef dout_prefix
 #define dout_prefix *_dout << "mgr " << __func__ << " "
